patch/kernel/rockchip64-dev_bak/RK3328-enable-1512mhz-opp.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
```

Comment:
It would be good to remove `rockchip64-dev_bak` folder and `*.orig` files before the merge ;p

Reply:
hmm fun that is still there.. I thought I already deleted it.